scripts/jenkins/ardana/ansible/deploy-caasp.yml

```diff
@@ -0,0 +1,55 @@
+---
+
+- name: Deploy CaaSP using heat templates
+  hosts: ardana-ccp-c0-m1-mgmt
```

Comment:
This playbook will be launched on the Jenkins worker node and it will use the hosts file created there (see the `ansible-playbook -v -i hosts` part). This inventory file only has one target - the deployer's floating IP - so you can use `hosts: hosts` here, the same as all other playbooks. 
By the way, this also means that all the tasks in this playbook will be executed on the deployer node, which is hopefully what you had in mind.

Reply:
yes that should be fine.